packages/test-utils/src/error-wrapper.js

```diff
@@ -1,56 +1,73 @@
 // @flow
 
 import { throwError } from 'shared/util'
+import { REF_SELECTOR } from 'shared/consts'
+
+const buildSelectorString = (selector: Selector) => {
+  if (selector.type === REF_SELECTOR) {
+    return `ref="${selector.value.ref}"`
+  }
+
+  if (typeof selector.value === 'string') {
+    return selector.value
+  }
+
+  return 'Component'
```

Comment:
I believe that we could use the component name at least in some cases here. However I'd like to keep that as a follow-up.